transit/tests/api_test/mixin/user_mixin.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import (
+    User,
+    Permission
+)
+from django.contrib.contenttypes.models import ContentType
+
+from transit.models import UserManager
+
+
+class UserMixin:
+    _TEST_USERNAME = "TestUser"
+    _TEST_PASSWORD = "TestPassword"
+    _TEST_DATA_USER = {
+        "username": _TEST_USERNAME,
+        "is_staff": True,
+        "password": _TEST_PASSWORD
+    }
+
+    def get_or_create_user(self):
+        user = User.objects.filter(username=self._TEST_USERNAME).first()
+        if user is None:
+            return self.__create_user()
+        return user
+
+    def add_permissions_to_manage_users(self):
```

Comment:
can you change it to more generic 
`add_user_permission(self, permission_name)`
so we'll be able in future for other permissions? 

If we want to add user all permissions from given proxy, then we could add one more method 
`add_user_proxy_permissions(self, proxy_model)`

Reply:
yes, of course, could be good. Thanks for this idea. I will implement this.